internals/cli/wait.go

```diff
@@ -31,7 +31,8 @@ var (
 )
 
 type waitMixin struct {
-	NoWait bool `long:"no-wait"`
+	NoWait       bool `long:"no-wait"`
```

Comment:
Per above, I think we should revert the changes to this file and use `Client.WaitChange` instead.